sechub-sereco/src/test/java/com/daimler/sechub/sereco/importer/SarifV1JSONImporterTest.java

```diff
@@ -0,0 +1,155 @@
+package com.daimler.sechub.sereco.importer;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.daimler.sechub.sereco.ImportParameter;
+import com.daimler.sechub.sereco.metadata.SerecoCodeCallStackElement;
+import com.daimler.sechub.sereco.metadata.SerecoMetaData;
+import com.daimler.sechub.sereco.metadata.SerecoSeverity;
+import com.daimler.sechub.sereco.metadata.SerecoVulnerability;
+import com.daimler.sechub.sereco.test.SerecoTestFileSupport;
+
+public class SarifV1JSONImporterTest {
+
+    private SarifV1JSONImporter importerToTest;
+    private String sarifBrakeman;
+    private String sarifEmptyResult;
+    private String sarifThreadflowsExample;
+    
+    @Before
+    public void before() {
+        importerToTest = new SarifV1JSONImporter();
+        sarifBrakeman = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_brakeman.json");
+        sarifEmptyResult = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_empty_results.json");
+        sarifThreadflowsExample = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_threadflows_example.json");
+    }
+
+    @Test
+    public void sarifreport_can_be_imported() {
```

Comment:
`sarifreport` in one word is hard to read. I would prefer `sarif_report`.

Reply:
done